packages/providers/fal/image-to-video/fal_image_to_video/models/base.py

```
@@ -244,4 +247,22 @@ def get_model_info(self) -> Dict[str, Any]:
+            elif "cost" in price:
+                price = price["cost"]
+            else:
+                # Use first numeric value found
+                for v in price.values():
+                    if isinstance(v, (int, float)):
+                        price = v
+                        break
+                else:
+                    raise ValueError(
+                        f"No numeric pricing tier found for model '{self.model_key}'"
+                    )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)